packages/frontend/src/routes/app/reports/[id]/page.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import { useParams, useNavigate } from 'react-router-dom'
+import { useQuery } from '@tanstack/react-query'
+import { QueryKeys } from '@/constants/queryKeys'
+import { ReportService } from '@/features/core/services/ReportService/ReportService'
+import { format } from 'date-fns'
```

Comment:
This project doesn't install `date-fns`. I suggest to use `dayjs` to format date.

Reply:
fixed